swift/example_code/iam/README.md

```diff
@@ -0,0 +1,71 @@
+# Amazon IAM code examples for the SDK for Swift
+## Overview
+This folder contains code examples demonstrating how to use the AWS SDK for
+Swift to use the Amazon Identity and Access Management (`IAM`). This README
+discusses how to run these examples.
+
+Amazon Identity and Access Management (`Amazon IAM`) lets you specify who and
+what can access services and resources on your AWS account, centrally manage
+fine-grained permissions, and analyze access to refine permissions across AWS.
```

Comment:
Consider updating the service description to match what's currently on the landing page: 
https://docs.aws.amazon.com/iam/